assessments/projects/cubefs/self-assessment.md

```diff
@@ -0,0 +1,235 @@
+# CubeFS Self-assessment
+This Self-assessment document has been created by students at NYU Tandon to help provide information regarding CubeFS, determining gaps in their security, and preparing any security

```

Comment:
Do we want to change this lead in?  I would think that it should mention the Security Pals effort.

Reply:
@JustinCappos do you have any suggestions for this?

Reply:
Please use: 

This assessment was created by community members as part of the [Security Pals](https://github.com/cncf/tag-security/issues/1102) process, and is currently pending changes from the maintainer team.

Reply:
Thank you @JustinCappos! The suggested change has been implemented. Please review.